go.mod

```diff
@@ -1,16 +1,16 @@
 module github.com/operator-framework/operator-registry
 
-go 1.22.0
+go 1.22.4
```

Comment:
Why not `go1.22.5` - same as toolchain?

Reply:
Honestly, because it's what resulted in `go get` and I presumed there might be a reason.  Retrospectively, I think that's wasteful behavior because if anything heeded to go version only, the toolchain would force downloading 1.22.5 anyway. 
Updated.

Reply:
Interestingly enough, `go mod tidy` with aligned go and toolchain versions deletes the toolchain statement. Makes sense, but could be a surprise if you didn't realize it.